_rules/button-accessible-name-97a4e1.md

```diff
@@ -50,55 +50,55 @@ There are no major accessibility support issues known for this rule.
 
 #### Passed Example 1
 
-Regular button.
+The [accessible name][] of `button` elements comes from it's content.
 
 ```html
 <button>My button</button>
 ```
 
 #### Passed Example 2
 
-Value attribute as the [accessible name][].
+The [accessible name][] of the `input` button comes from it's `value` attribute.
 
 ```html
 <input type="submit" value="Submit" />
 ```
 
 #### Passed Example 3
 
-`aria-label` for the [accessible name][].
+When there is no text content, a `button` can be given an [accessible name][] with the `aria-label` attribute.
```

Comment:
Technically, `aria-label` can also be used when there is text content (and trumps it).
```suggestion
The [accessible name][] of the `button` comes from its `aria-label` attribute.
```